fbw-a380x/src/wasm/systems/a380_systems/src/lib.rs

```diff
@@ -258,18 +256,18 @@ impl Aircraft for A380 {
             [self.lgcius.lgciu1(), self.lgcius.lgciu2()],
         );
 
-        self.engines_flex_physics.update(context);
-        self.elevators_flex_physics.update(
+        self.cds.update();
+
+        self.egpwc.update(&self.adirs, self.lgcius.lgciu1());
+
+        self.structural_flex.update(
```

Comment:
Admittedly I'm not that familiar with the code at such a low level, but shouldn't ``self.structural_flex.update()`` belong in ``update_before_power_distribution()`` and not ``update_after_power_distribution()`` ?

Reply:
No real rule about this when it comes to something that doesn't have electrical, but as it has dependencies on hydraulics which is updated after elec, better to put it there so it's updated at frame N with latest hydraulic values